build_wheel.sh

```diff
@@ -3,8 +3,23 @@
 THIS_SCRIPT=$0
 SCRIPT_DIR=$(dirname -- "$(readlink -f -- "$THIS_SCRIPT")")
 
+YEAR="${EBVERSIONGENTOO:-2017}"
```

Comment:
Should we set it to 2023 by default if not defined?

Reply:
I think we could remove 2017 from the build script, esp since we are talking about deprecating the older stacks.

Reply:
I think so too, but better in a different PR?

Reply:
yes indeed, a different PR would be better :)